api/controllers/routes.go

```diff
@@ -20,10 +20,12 @@ func NewRouter() (router *mux.Router) {
 	router.HandleFunc("/apps/{app}", api("app.delete", AppDelete)).Methods("DELETE")
 	router.HandleFunc("/apps/{app}/builds", api("build.list", BuildList)).Methods("GET")
 	router.HandleFunc("/apps/{app}/builds", api("build.create", BuildCreate)).Methods("POST")
+	router.HandleFunc("/apps/{app}/builds/import", api("build.import", BuildImport)).Methods("POST")
```

Comment:
All of the build paths return a Build object which itself has a Release attribute (that's how things like `convox build --promote` work)